scripts/front/reorder-admin-translation-files.js

```diff
@@ -0,0 +1,65 @@
+const { join } = require('path');
+const fs = require('fs-extra');
+const { promisify } = require('util');
+const glob = promisify(require('glob').glob);
+
+async function orderTrads({ mainTranslationFile, translationFiles }) {
+  const data = await fs.readJSON(mainTranslationFile);
+  const orderedData = Object.keys(data)
+
+    .sort()
+    .reduce((acc, current) => {
+      acc[current] = data[current];
+
+      return acc;
+    }, {});
+
+  await fs.writeJSON(mainTranslationFile, orderedData, { spaces: 2 });
+
+  await Promise.all(
+    translationFiles.map(async trad => {
```

Comment:
you should not use an async function directly in a map as it will throw an unhandledRejectionError if the function fails somehow in the future